lib/flutter_svg_provider.dart

```diff
@@ -111,20 +111,24 @@ class Svg extends ImageProvider<SvgImageKey> {
   }
 
   static Future<ImageInfo> _loadAsync(SvgImageKey key) async {
-    final String rawSvg = await _getSvgString(key);
+    final rawSvg = await _getSvgString(key);
```

Comment:
What is the reason for the change here?

Reply:
Unnecessary type annotation on a local variable. ([Documentation](https://dart.dev/tools/linter-rules/omit_local_variable_types))